mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraAttachmentModule.java

```diff
@@ -48,7 +50,14 @@ public CassandraAttachmentModule() {
                     .addColumn(CassandraAttachmentTable.TYPE, text())
                     .addColumn(CassandraAttachmentTable.SIZE, bigint())
                     .withOptions()
-                    .comment("Holds attachment for fast attachment retrieval")));
+                    .comment("Holds attachment for fast attachment retrieval")),
+            new CassandraTable(CassandraAttachmentMessageIdTable.TABLE_NAME,
+                SchemaBuilder.createTable(CassandraAttachmentMessageIdTable.TABLE_NAME)
+                    .ifNotExists()
+                    .addPartitionKey(CassandraAttachmentMessageIdTable.ATTACHMENT_ID, text())
+                    .addColumn(CassandraAttachmentMessageIdTable.MESSAGE_IDS, set(text()))
```

Comment:
I prefer : 

```
.addClusteringColumn(CassandraAttachmentMessageIdTable.MESSAGE_ID, text());
```

Rationals: *collections are advanced features*. If we can avoid them with the **basic data modelling** of Cassandra, then we should. (Note: in this case, the underlying structure would be the exact same)

Also please, be *consistent* with the code base. See **CassandraMailboxRecentsModule**, **CassandraFirstUnseenModule**, **CassandraDeletedMessageModule** for instance. So far we valued and enforced clustering columns over collections.

Reply:
done